README.md

```diff
@@ -1,12 +1,14 @@
-# eth-faucet
+# lsk-faucet
 
-[![Build](https://img.shields.io/github/actions/workflow/status/chainflag/eth-faucet/build.yml?branch=main)](https://github.com/chainflag/eth-faucet/actions/workflows/build.yml)
-[![Release](https://img.shields.io/github/v/release/chainflag/eth-faucet)](https://github.com/chainflag/eth-faucet/releases)
-[![Report](https://goreportcard.com/badge/github.com/chainflag/eth-faucet)](https://goreportcard.com/report/github.com/chainflag/eth-faucet)
-[![Go](https://img.shields.io/github/go-mod/go-version/chainflag/eth-faucet)](https://go.dev/)
-[![License](https://img.shields.io/github/license/chainflag/eth-faucet)](https://github.com/chainflag/eth-faucet/blob/main/LICENSE)
+[![Report](https://goreportcard.com/badge/github.com/liskhq/lsk-faucet)](https://goreportcard.com/report/github.com/liskhq/lsk-faucet)
+[![Go](https://img.shields.io/github/go-mod/go-version/liskhq/lsk-faucet)](https://go.dev/)
+[![License](https://img.shields.io/github/license/LiskHQ/lsk-faucet)](https://github.com/liskhq/lsk-faucet/blob/main/LICENSE)
+![GitHub repo size](https://img.shields.io/github/repo-size/liskhq/lsk-faucet)
+![GitHub issues](https://img.shields.io/github/issues-raw/liskhq/lsk-faucet)
+![GitHub closed issues](https://img.shields.io/github/issues-closed-raw/liskhq/lsk-faucet)
 
-The faucet is a web application with the goal of distributing small amounts of Ether in private and test networks.
+LSK faucet is a web application to get sepolia Lisk (LSK) in order to test and troubleshoot your decentralized application or protocol before going live on Lisk mainnet.
+This faucet is designed to provide LSK test tokens to the developers who need to test their smart contracts and interact with the blockchain.
```

Comment:
This statement is incorrect since the transaction token on Lisk L2 (both Mainnet & Sepolia Testnet) is bridged ETH. Let's please update/remove it.
@shuse2 could you please suggest why would someone need LSK tokens on the testnet, other than for governance which in my understanding is only used by our contracts?
```suggestion
This faucet is designed to provide LSK test tokens to the developers who need to test their smart contracts and interact with the blockchain.
```

Reply:
I removed the statement.